build_all.sh

```diff
@@ -17,6 +17,24 @@ set -o pipefail
 # shellcheck source=common_functions.sh
 source ./common_functions.sh
 
+# summary table array
+summary_table_file="./.summary_table"
```

Comment:
You can define it here and export it so that you dont have to redefine in `build_latest.sh`. In `build_latest.sh` you can check if it is not set and set it only if needed.

Reply:
Made changes accordingly. Had also exported `root_dir` in `build_all.sh`